CryptoPrices/Sources/Domain/Sources/Entities/TimeFilter.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TimeFilter.swift
+//  Domain
+//
+//  Created by Minh Pham on 27/12/2022.
+//
+
+public enum TimeFilter {
+    case oneDay
```

Comment:
```suggestion
public enum TimeFilter {

    case oneDay
```

Reply:
Fixed in [4338b7e](https://github.com/nimblehq/nimble-crypto-ios/pull/75/commits/4338b7e9fe2efa557c9c644bfc0289722e2561ce) 🚀